package.json

```diff
@@ -49,7 +50,7 @@
     "webpack-dev-server": "^1.8.2"
   },
   "peerDependencies": {
-    "react": ">=0.13.2 || ^0.14.0-rc1 || ^15.0.0-rc"
```

Comment:
To be pedantic, 15 stable is *already* allowed. `^15.0.0-rc` includes any stable `15` version.